CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## 0.1.8
+
+### Preview features
```

Comment:
Do we want to start including the blog post here in a summary? Cross linking between the blog post and changelog might be useful.

Reply:
Yeah good call.